service/routers/users.js

```diff
@@ -0,0 +1,50 @@
+import { Router } from 'express'
+import { User } from '../models/User.js'
+
+const usersRouter = Router()
+
+const getUsers = async (req, res) => {
+  if(req.user.role == "Admin" || req.user.role == "SuperUser") {
+    const users = await User.find({})
+    res.send(users)
+  } else {
+    res.status(404).send('only admin and superuser can access this route')
+  }
+}
+
+const getUsersById = async (req, res) => {
+  if(req.user.role == "Admin" || req.user.role == "SuperUser") {
+    const user = await User.findById(req.params.id)
+  res.send(user)
+  } else {
+    res.status(404).send('only admin and superuser can access this route')
+  }
+}
+
+const updateUser = async (req, res) => {
+  if(req.user.role == "Admin" || req.user.userId == req.params.id) {
```

Comment:
This code is incomplete. Please re-check the assignment criteria and make sure you're handling all the roles and cases here.